client/dashboard/leaderboard/index.js

```diff
@@ -0,0 +1,96 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+import { compose } from '@wordpress/compose';
+import { get } from 'lodash';
+import PropTypes from 'prop-types';
+
+/**
+ * WooCommerce dependencies
+ */
+import { Card, EmptyTable, TableCard } from '@woocommerce/components';
+
+/**
+ * Internal dependencies
+ */
+import ReportError from 'analytics/components/report-error';
+import { getReportTableData } from 'store/reports/utils';
+import withSelect from 'wc-api/with-select';
+import './style.scss';
+
+export class Leaderboard extends Component {
+	render() {
+		const { getHeadersContent, getRowsContent, isRequesting, isError, items, title } = this.props;
+		const data = get( items, [ 'data' ], [] );
+		const rows = isRequesting || isError ? [] : getRowsContent( data );
```

Comment:
Is the `isRequesting || isError ? [] : ` portion redundant since we check for errors and requesting below?  Or is there a chance that malformed data could be returned for the rows?